samples/bluetooth/broadcast_audio_sink/overlay-nrf5340_cpuapp_bt_zephyr_ctrl.conf

```diff
@@ -0,0 +1,6 @@
+# Include this overlay config when using the
+# Zephyr BT controller (hci_rpmsg) on nRF5340
+
+CONFIG_BT_HCI_ACL_FLOW_CONTROL=n
+CONFIG_BT_HCI_VS_EXT=n
+CONFIG_BT_TINYCRYPT_ECC=y
```

Comment:
I was the one suggesting to move it to an overlay file as its stricktly not required if another controller supports different things - but I might be wrong? On the other hand, in a pure Zephyr context (using current hci_rpmsg) it would always be needed for the board, I guess?